cloudformation/cloudformation.yaml

```diff
@@ -264,7 +264,7 @@ Resources:
           S3Key: !Sub membership/${Stage}/fulfilment-lambdas/fulfilment-lambdas.zip
         MemorySize: 512
         Runtime: nodejs14.x
-        Timeout: 300
+        Timeout: 0
```

Comment:
what does 0 mean? 
<img width="674" alt="image" src="https://github.com/guardian/fulfilment-lambdas/assets/7304387/d3cfd0cb-3a32-4b49-ab86-de9fda7be858">
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-timeout

Reply:
oh, i didn't mean to change that! did i do a find and replace without meaning to 🤔  i'll revert that!!!